### ISSUE TITLE
showInHand or showInInventory not working

### ISSUE THREAD

**Issue by stijnb1234** [InventivetalentDev/MapManager#46]
## What steps will reproduce the problem?  
1. I run wrapper.getController().showInHand(p, true);
2. The user get's an empty image

## What were you expecting to happen? What happened instead?  
A good image is shown

## What version of the plugin are you using? *Type /version &lt;Plugin Name&gt;*  
1.7.5-SNAPTSHOT

## What Spigot version are you using? *Type /version*  
1.14.4-R0.1-SNAPSHOT [PaperSpigot build 237]

## What plugins are you using? *Type /plugins*  
![image](https://user-images.githubusercontent.com/27051051/72661653-fab55e80-39dc-11ea-9ed7-f0cd48593d2a.png)

The image (ArrayImage) is okay. 
![image](https://user-images.githubusercontent.com/27051051/72661662-32240b00-39dd-11ea-9f89-ee5695ca7ec6.png)

But it's not shown in the inventory, and it says "Unkown map"
![image](https://user-images.githubusercontent.com/27051051/72661656-0d2f9800-39dd-11ea-83a4-f410281b57a3.png)

![image](https://user-images.githubusercontent.com/27051051/72661685-87f8b300-39dd-11ea-8436-a54bc5588635.png)

**Comment by InventivetalentDev** [comment]
Would you mind attaching a snippet of your code (for both the itemframe and inventory ones)?

**Comment by stijnb1234** [comment]
```java
public static void render() {
    //AFRender is my object where I store the MapWrapper
    AFRender render = new AFRender();

    BufferedImage img = ...;
    MapWrapper wrapper = mapManager.wrapImage(img);

    render.setWrapper(wrapper);

    for (Player p: Bukkit.getServer().getOnlinePlayers()) {
        showForPlayer(p, /* I get the frame from another object */ , render);
    }
}

public static void showForPlayer(Player p, ItemFrame frame, AFRender render) {
    MapController controller = render.getWrapper().getController();
    controller.addViewer(p);
    controller.sendContent(p);
    controller.showInFrame(p, frame, true);
}

public static void showInInv(Player p, AFRender render) {
    render.getWrapper().getController().showInHand(p, true);
}
```

**Comment by InventivetalentDev** [comment]
Fixed in 1.7.6-SNAPSHOT :)
